product-service/handler.js

```diff
@@ -0,0 +1,60 @@
+const mock = require('./mock.json');
+// import mock from './mock.json';
+module.exports.getProductsList = async (event) => {
+  try {
+    const res = await Promise.resolve(mock);
+    console.log('res', res);
+    return {
+      statusCode: 200,
+      headers: {
+        'Access-Control-Allow-Methods': '*',
+        'Access-Control-Allow-Headers': '*',
+        'Access-Control-Allow-Origin': '*'
+      },
+      body: JSON.stringify(res),
+    };
+  } catch (error) {
+    return {
+      statusCode: 500,
+      headers: {
+        'Access-Control-Allow-Methods': '*',
+        'Access-Control-Allow-Headers': '*',
+        'Access-Control-Allow-Origin': '*'
+      },
+      body: JSON.stringify({ message: error.message })
+    };
+  }
+
+  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
+  // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
+};
+
+module.exports.getProductsById = async (event) => {
+  const id = event.pathParameters.id;
```

Comment:
You can also destructure it like
```suggestion
  const { id } = event.pathParameters;
```